doc/release-notes.md

```diff
@@ -274,6 +277,14 @@ To enable block pruning set `prune=<N>` on the command line or in
 `bitcoin.conf`, where `N` is the number of MiB to allot for
 raw block & undo data.
 
+A value of 0 disables pruning. The minimal value above 0 is 550. Your
+wallet is as secure with high values as it is with low ones. Higher
+values merely reduce the network traffic in case of reorganization of
+the blockchain. In future releases, a higher value may also help the
+network as a whole: The stored blocks could be served to other nodes.
+Currently, nodes with pruning enabled do not serve block data at all
+and thus could be considered as "leechers".
```

Comment:
I think we should remove that last sentence for now, as we're not in a position where "leaching" of blockchain data distribution bandwidth matters.